README.md

```diff
@@ -15,7 +15,16 @@ Turn Twitch chatter into Python events.
 
 ## Usage
 
-### 1. Create an Observer
+### 1. Installation
+
+To install twitch-observer you need to download this GitHub repo and call `pip` in your shell of choice:
```

Comment:
You are right, for the target audience this is to specific.
I also added a note for PyPI support in the future.